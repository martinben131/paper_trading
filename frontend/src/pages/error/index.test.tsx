import React from "react";
import { shallow } from "enzyme";
import Error from "./index";

test("Renders error page correctly", () => {
  const component = shallow(<Error />);
  expect(component).toMatchSnapshot();
});

test("Page shows a 404 error", () => {
  const component = shallow(<Error />);
  expect(component.text()).toContain("404");
});
