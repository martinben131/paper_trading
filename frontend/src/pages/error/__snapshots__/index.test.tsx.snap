// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders error page correctly 1`] = `
<Fragment>
  <h1
    className="bp3-heading"
  >
    Uh-oh 
    <span
      aria-label="Crying Face"
      role="img"
    >
      ðŸ˜¢
    </span>
  </h1>
  <br />
  <h1
    className="bp3-heading"
  >
    404 - Page not found
  </h1>
  <br />
  <Blueprint3.Button
    intent="primary"
    large={true}
    onClick={[Function]}
    rightIcon="home"
    text="Go home"
  />
</Fragment>
`;
